src/main/java/com/google/cloud/resourcemanager/ConstraintInfo.java

```diff
@@ -0,0 +1,314 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.resourcemanager;
+
+import com.google.api.services.cloudresourcemanager.model.BooleanConstraint;
+import com.google.common.base.Function;
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Optional;
+import java.io.Serializable;
+import java.util.Objects;
+
+/** A Google Cloud Resource Manager constraint metadata object. */
+public class ConstraintInfo implements Serializable {
+  static final Function<
+          com.google.api.services.cloudresourcemanager.model.Constraint, ConstraintInfo>
+      FROM_PB_FUNCTION =
+          new Function<
+              com.google.api.services.cloudresourcemanager.model.Constraint, ConstraintInfo>() {
+            @Override
+            public ConstraintInfo apply(
+                com.google.api.services.cloudresourcemanager.model.Constraint pb) {
+              return ConstraintInfo.fromPb(pb);
+            }
+          };
+  static final Function<
+          ConstraintInfo, com.google.api.services.cloudresourcemanager.model.Constraint>
+      TO_PB_FUNCTION =
+          new Function<
+              ConstraintInfo, com.google.api.services.cloudresourcemanager.model.Constraint>() {
+            @Override
+            public com.google.api.services.cloudresourcemanager.model.Constraint apply(
+                ConstraintInfo constraintInfo) {
+              return constraintInfo.toPb();
+            }
+          };
+
+  private static final long serialVersionUID = 9148970963697734236L;
```

Comment:
Move it to line 27, just after the class parenthesis starts. Applies to below classes

Reply:
done